app/views/plate_creation/donor_pooling_plate.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= page('donor-pooling-page') do %>
+  <%= content do %>
+    <%= card id: 'main-content' do %>
+      <div>
+        <h3>Instructions</h3>
+        <p>This step uses multiple source plates for pooling together into a destination plate.</p>
+        <p>This process checks that the source plates are compatible and can be used together.</p>
+        <p>Scan the source plate barcodes into the panel on the right of the screen (the order doesn't matter) and click <i>Create Plate</i>.</p>
+      </div>
+    <% end %>
+  <% end %>
+  <%= sidebar do %>
+    <%= card title: 'Add plates to pool', without_block: true do %>
+      <div id="validation_report"></div>
+      <div id="add-plates-instructions-block" class="card-body" >
+        <div class="instructions">
+          <p>Scan in up to <% @labware_creator.number_of_source_plates %> plates.</p>
+        </div>
+      </div>
+    <% end %>
+    <div class="card-body">
+      <%= form_for(
+        @labware_creator,
+        url: limber_plate_children_path(@labware_creator.parent_uuid),
+        as: :plate,
+        html: { method: :post }) do |form| %>
+
+        <%= form.hidden_field :parent_uuid %>
+        <%= form.hidden_field :purpose_uuid %>
+        <% @labware_creator.number_of_source_plates.times do |index| %>
+          <div class="labware-container scan-labware form-group row">
+            <label for="plate[barcodes][<%= index %>]"
+                   class="plate-label col-md-2" >Source Plate <%= index + 1 %></label>
+            <input type="text"
+                   id="plate[barcodes][<%= index %>]"
+                   name="plate[barcodes][]"
+                   class="form-control col-md-8"
+                   required="required"/>
+          </div>
+        <% end %>
+        <%= form.submit 'Create Plate', class: 'btn btn-success btn-lg', id: 'create-labware' %>
+    </div>
+  <% end %>
+<% end %>
```

Comment:
I left a comment on Andrew's PR about whether we could provide a generic / configurable way to upload a tube rack scan as part of a transfer (https://github.com/sanger/limber/pull/1586#discussion_r1511031374). I wonder if in the future we could do something similar with scanning extra barcodes. Seems like a common requirement, to scan in barcodes as part of the transfer process. You could configure how many you want to ask for. 

Not asking for this as part of this PR, just an idea for the future!

Reply:
Yes, form fields can be generated in a loop like here up to a configured number and parsed accordingly. However, there is still a need to distinguish between contingency and sequencing racks in the referenced PR.